package-lock.json

```diff
@@ -0,0 +1,1855 @@
+{
```

Comment:
I believe that we should move package-lock json to gitignore here, it's enough to rely simply on package.json dependencies without `^`...

Reply:
I believe we had an agreement internally we'll use both dependency locking and commit the `package-lock.json`.
@bariscicek any thoughts here?

Reply:
Baris suggested it's better to remove the lock file since it's a facility.